packages/tabs/src/Tabs.tsx

```diff
@@ -231,25 +231,31 @@ function Tabs({
     }
 
     const isTabSelected = index === selected;
-    const { disabled, onClick } = child.props;
+    const { disabled, onClick, onKeyDown, className, ...rest } = child.props;
 
     const tabProps = {
-      ...child.props,
-      disabled,
       as,
+      disabled,
       darkMode,
       isAnyTabFocused,
-      onKeyDown: handleArrowKeyPress,
-      className: cx({
-        [modeColors[mode].activeStyle]: isTabSelected,
-        [cx(modeColors[mode].disabledColor, disabledStyle)]: disabled,
-      }),
+      className: cx(
+        {
+          [modeColors[mode].activeStyle]: isTabSelected,
+          [cx(modeColors[mode].disabledColor, disabledStyle)]: disabled,
+        },
+        className,
+      ),
+      onKeyDown: (event: KeyboardEvent) => {
+        handleArrowKeyPress(event);
+        onKeyDown?.(event);
```

Comment:
just to be a bit more consistent with the ordering of the callbacks
```suggestion
        onKeyDown?.(event);
        handleArrowKeyPress(event);
```